libs/blocks/aside/aside.css

```diff
@@ -425,15 +631,29 @@
     bottom: 0;
   }
 
-  .aside.split-right .split-image img {
+  .aside.split .tablet-wide img,
+  .aside.split .tablet-wide video {
+    aspect-ratio: 16/9;
+  }
+  
+  .aside.split .tablet-standard img,
+  .aside.split .tablet-standard video {
+    aspect-ratio: 4/3;
```

Comment:
IMO these aspect ratio values could be added to our variables so they can be used in other blocks that offer up video rendering. Thoughts?

Reply:
@Sartxi I have made the requested change and pushed the constants to variables to be available for use globally.

Reply:
okay great!